lib/routes/account.js

```diff
@@ -560,6 +612,8 @@ module.exports = function (
             } else {
               response.verificationReason = 'login'
             }
+          } else {
+            recordSecurityEvent() // don't block response recording db event
```

Comment:
Would it make sense to log a security event, even if doSigninConfirmation is true.?